pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md

```diff
@@ -8,10 +8,15 @@ Model that is able to detect more or less horizontal text with high speed on CPU
 
 ## Training pipeline
 
-### 0. Change a directory in your terminal to object_detection.
+### 0. Change a directory in your terminal to object_detection and activate venv.
```

Comment:
Regarding this change: activating of venv is mentioned here: https://github.com/openvinotoolkit/training_extensions/tree/develop/pytorch_toolkit/object_detection/readme.md

If you would like to apply your change please do it for all templates

Reply:
Added info about virtual environment activation to other templates